docs/other/install-cvmfs-stratum1.md

```diff
@@ -0,0 +1,338 @@
+
+## Users and Groups
+
+If your machine is also going to be a repository server like the OSG GOC, the installation will create one user unless it already exists:
+
+<span class="twiki-macro STARTSECTION">Users</span>
+
+| User    | Comment                   |
+|:--------|:--------------------------|
+| `cvmfs` | CernVM-FS service account |
+
+<span class="twiki-macro ENDSECTION">Users</span>
+
+A repository server installation will also create a cvmfs group and default the cvmfs user to that group.
+
+In addition, if the fuse rpm is not for some reason already installed, installing the repository server packages will also install fuse and that will create another group:
+
+<span class="twiki-macro STARTSECTION">Groups</span>
+
+| Group   | Comment                   | Group members |
+|:--------|:--------------------------|:--------------|
+| `cvmfs` | CernVM-FS service account | none          |
+| `fuse`  | FUSE service account      | `cvmfs`       |
+
+<span class="twiki-macro ENDSECTION">Groups</span>
+
+# Install Instructions
+
+All CVMFS Stratum 1s require cvmfs-server software and apache (httpd). It is highly recommended to also install frontier-squid and frontier-awstats on the same machine to be able to easily join the WLCG [MRTG](http://wlcg-squid-monitor.cern.ch/snmpstats/indexcvmfs.html) and [awstats](http://wlcg-squid-monitor.cern.ch/awstats/cvmfs.html) monitoring systems. The recommended configuration for frontier-squid below does not do any caching, it is just for monitoring.
+
+## Installing cvmfs-server and httpd
+
+The OSG GOC Stratum 1 has to function as a repository server in addition to serving repository replications; most Stratum 1s serve only replications. Serving repositories is done quite differently on Redhat EL5 and EL6 systems. Instructions are also provided for how to install cvmfs-server on Stratum 1s that do not have to be repository servers, which is the same on both versions of operating systems. Choose one of the following three sections.
+
+### Installing a CVMFS repository server
+
+#### Installing CVMFS repository server on RHEL5
+
+Redhat EL5-based systems that are not Scientific Linux-based should first build the [SL5 aufs source rpm](http://ftp.scientificlinux.org/linux/scientific/5x/SRPMS/SL/aufs-0.20090202.cvs-6.sl5.src.rpm) and install the resulting rpm in their own yum repository.
+
+Follow these instructions to install:
+
+```
+[root@client ~] $ rpm -i http://cvmrepo.web.cern.ch/cvmrepo/yum/cvmfs/EL/5/x86_64/cvmfs-release-2-4.el5.noarch.rpm
```

Comment:
Please convert the prompts using the following table: https://opensciencegrid.github.io/technology/software/markdown-migration/#fixing-root-and-user-prompts